src/shared/components/challenge-listing/Filters/FiltersPanel/index.jsx

```diff
@@ -53,6 +53,17 @@ export default function FiltersPanel({
 
   const mapOps = item => ({ label: item, value: item });
 
+  let filterStateTags = null;
+  if (filterState.tags) {
+    filterStateTags = Array.isArray(filterState.tags) ? filterState.tags.join(',') :
+      Object.keys(filterState.tags).map(key => filterState.tags[key]).join(',');
+  }
```

Comment:
1. For such checks we prefer to use lodash functions.
2. Is really possible that we can get either array either object here? It looks strange.